src/parameterizations/vertical/MOM_vert_friction.F90

```diff
@@ -914,12 +1073,13 @@ subroutine vertvisc_coef(u, v, h, forces, visc, dt, G, GV, US, CS, OBC)
     if (do_any_shelf) then
       do K=1,nz+1 ; do I=Isq,Ieq ; if (do_i_shelf(I)) then
         CS%a_u(I,j,K) = min(a_cpl_max, forces%frac_shelf_u(I,j)  * a_shelf(I,K) + &
-                                       (1.0-forces%frac_shelf_u(I,j)) * a_cpl(I,K))
+                                       (1.0-forces%frac_shelf_u(I,j)) * a_cpl(I,K) + a_cpl_gl90(I,K))
```

Comment:
Whenever we add 3 real numbers, we try to add parentheses so that different compilers and compiler settings will give the same answers.  Please add parentheses to specify the order with which the 3 values are being added both here and later on line 1267.

Reply:
Makes sense - now fixed!